### PR TITLE
add release manage docs

### PR DESCRIPTION
**Pull request by Shelley-BaoYue**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

docs update

* **What is the new behavior (if this is a feature change)?**

add release manage docs

### CODE REVIEW

File: docs/community/release.md [hunk 1]
```diff
@@ -1,6 +1,32 @@
 ---
-title: Release Manager
+title: Release Manage
```
Comment (fisherxu) [inline docs/community/release.md:2]:
```suggestion
title: Release Management
```

File: docs/community/release.md [hunk 2]
```diff
@@ -1,6 +1,32 @@
...
 sidebar_position: 5
 ---
 
-tbd...
+KubeEdge releases currently happen approximately four times per year.
```
Comment (fisherxu) [inline docs/community/release.md:6]: every three months should be more accurate?

File: docs/community/release.md (hunk 1 above)
Comment (Shelley-BaoYue) [reply docs/community/release.md]: done.

File: docs/community/release.md (hunk 2 above)
Comment (Shelley-BaoYue) [reply docs/community/release.md]: > every three months should be more accurate?

done.

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**